css/index.css

```diff
@@ -1 +1,377 @@
+  height: 40px;
+  /*background: lightgrey; */
+}
+
+.header {
+  border: 1px solid purple;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 60px 0;
+  margin-top: .9rem;
+}
+
+.banner {
+  border: 1px solid brown;
+  display: flex;
+  height: 360px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-left: 12rem;
+  padding-bottom: 1rem;
+}
+
+h1 {
+  font-size: 7.4rem;
+  font-family: 'Bangers', sans-serif;
+  text-align: center;
+  margin: 0;
+  line-height: 1;
+}
+
+button {
+
+  text-align: center;
+  vertical-align: center;
+  font-size: 1.7rem;
+  border: 1px solid black;
+  width: 55%;
+  padding: .6rem 0;
+  margin-top: 2.5rem;
+  margin-right: 1rem;
+
+}
+
+button:hover {
+  background: lightgray;
+}
+
+.headerIMG {
+  border: 1px solid brown;
+  max-width: 38rem;
+  width: 100%;
+}
+
+.main {
+  border-top: .2rem solid black;
+  border-bottom: .2rem solid black;
+  padding: 3.5rem 0;
+  margin: 1.2rem 0;
+  font-size: 1.7rem;
+}
+
+
+.top {
+  display: flex;
+  justify-content: space-between;
+}
+
+.top .top-content{
+  border: 1px solid blue;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  width: 49.5%;
+}
+
+.middleIMG{
+  max-width: 88rem;
+  width: 100%;
+  padding: 2.9rem 0;
+
+}
+
+.bottom{
+  border: 1px solid orange;
+  display: flex;
+  justify-content: space-between;
+}
+
+.bottom-content {
+  border: 1px solid blue;
+  display: flex;
+  flex-direction: column;
+  width: 32%;
+}
+
+footer {
+  border: 1px solid green;
+  padding-top: 3rem;
+}
+
+footer address {
+  font-size: 1.6rem;
+}
+
+footer address h2 {
+  border: 1px solid rebeccapurple;
+}
+
+footer address .phone {
+  padding: 1.2rem 0;
+}
+
+footer address .email {
+  padding: .5rem 0;
+}
+
+footer .copyright {
+  border: 1px solid blue;
+  text-align: center;
+  font-size: 1.45rem;
+  font-family: 'Titillium Web', sans-serif;
+  padding: 3.8rem 0;
+}
+
+
+
+/* ######    BREAK 800px    #########  */
+@media (max-width: 800px) {
+
+  body {
+    border: 1px solid purple;
+   /* width: 800px; */
+   margin: 0 auto;
+  }
+
+  .nav-container {
+    border: 1px solid green;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+  }
+
+  .nav-container .navbar_top {
+    border: 1px solid hotpink;
+    padding: 2rem;
+    display: flex;
+    justify-content: space-between;
+    width: 95%;
+    margin: 1rem;
+  }
+
+  .nav-container .navbar_top a {
+    font-size: 1.5rem;
+  }
+
+  .logoIMG {
+    border: 1px solid blue;
+    margin: 5px auto;
+    order: -1;
+  }
+
+  .headerIMG {
+    display: none;
+  }
+
+  .header {
+    border: 5px solid purple;
+    justify-content: center;
+    margin: 0;
+    padding: 0;
+  }
+
+  .banner {
+    margin: 0;
+    padding: 0;
+  }
+
+  .main {
+  }
+
+  .top .top-content{
+    width: 48%;
+
+  }
+
+  .bottom .bottom-content {
+    width: 30%;
+  }
+
+
+}
+
+
+@media (max-width: 500px) {
+
```

Comment:
looks like you kind of ran out of steam down here or you haven't pushed the latest. I assume this is still a work in progress given all the guide borders.